lte/gateway/c/core/oai/test/amf/amf_app_test_util.cpp

```diff
@@ -647,10 +647,7 @@ bool check_ue_context_state(amf_ue_ngap_id_t ue_id,
                             m5gcm_state_t expected_cm_state,
                             n2cause_e expected_ue_context_rel_cause) {
   m5gmm_state_t mm_state;
-  int rc = RETURNerror;
-
-  rc = amf_get_ue_context_mm_state(ue_id, &mm_state);
-  if (rc != RETURNok) {
+  if (amf_get_ue_context_mm_state(ue_id, &mm_state) != RETURNok) {
```

Comment:
Add an error log. Helps in debugging

Reply:
"lte/gateway/c/core/oai/test/amf/amf_app_test_util.cpp" this file is part of unit test framework and logging functionalities are not available while running the 5G unit test framework.

Reply:
@Akshayp77 : Can you please check "mme_app_task/mme_app_test.cpp" -> OAILOG_INIT. If it helps in any ways.

Reply:
Error logs are added.